iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/NodeBreakerVoltageLevel.java

```diff
@@ -200,6 +200,8 @@ public Switch add() {
             }
             SwitchImpl aSwitch = new SwitchImpl(NodeBreakerVoltageLevel.this, id, getName(), kind, open, retained, fictitious);
             getNetwork().getIndex().checkAndAdd(aSwitch);
+            graph.addVertex(node1);
+            graph.addVertex(node2);
```

Comment:
Ok for me, but we can wonder if it should not be in the addEdge method, after all.

Reply:
Calling `addVertex` in `addEdge` will introduce a lot of changes in the bahavior of the graph. It could change a lot of thing in the Bus/Breaker topology. As the issue is in the Node/Breaker topology, I prefered do like this, but this choice could be discussed.